pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `allOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `anyOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `const`                     | Validation    | 06                 | No      | No                                                                                           |
+| `contains`                  | Core          | 06                 | No      | No                                                                                           |
+| `contentEncoding` (legacy)  | Core          | 01                 | 04      | Replaced by `media`.`binaryEncoding`                                                         |
+| `media`                     | Hyper-Schema  | 04                 | 07      | Replaced `contentEncoding` (legacy) and replaced by `contentEncoding` and `contentMediaType` |
+| `contentEncoding`           | Validation    | 07                 | No      | Replaced `media`.`binaryEncoding`                                                            |
+| `contentMediaType`          | Validation    | 07                 | No      | Replaced `media`.`type`                                                                      |
+| `contentSchema`             | Validation    | 2019-09            | No      | No                                                                                           |
+| `definitions`               | Validation    | 04                 | 2019-09 | Replaced by` $defs`                                                                          |
+| `default`                   | Validation    | 01                 | No      | No                                                                                           |
+| `dependencies`              | Validation    | 03                 | 2019-09 | Replaced `requires` and replaced by `dependentSchemas` and `dependentRequired`               |
+| `dependentRequired`         | Validation    | 2019-09            | No      | Partially replaced `dependencies`                                                            |
+| `dependentSchemas`          | Core          | 2019-09            | No      | Partially replaced `dependencies`                                                            |
+| `deprecated`                | Validation    | 2019-09            | No      | No                                                                                           |
+| `disallow`                  | Core          | 01                 | 04      | Replaced by `not`                                                                            |
+| `divisibleBy`               | Validation    | 02                 | No      | Replaced by `multipleOf`                                                                     |
```

Comment:
This keyword was removed when it was replaced by `multipleOf`

Reply:
It was removed in draft-04

Reply:
Noted and adjusted.